ruma-events/benches/event_deserialize.rs

```diff
@@ -1,18 +1,18 @@
-// In order to run use either `cargo bench` or to save if you
-// want to save the result to compare branches use
-// `cargo bench --bench event_deserialize -- --save-baseline test-output`
-// Since we are in a workspace the default `cargo bench` still picks up
-// the args passed to it to avoid this use the above command.
+// `cargo bench` works but if you use `cargo bench -- --save-baseline <name>`
```

Comment:
Please punctuate your sentences correctly!

```suggestion
// `cargo bench` works, but if you use `cargo bench -- --save-baseline <name>`
```